index.html

```diff
@@ -0,0 +1,131 @@
+    </p>
+    
+    <!--Used ordered list and abbreviation tags. Also added subscript and suprscipt tags-->
+    <ol>
+        <li>Turning H<sub>2</sub>O into wine, and the health benefits of Resveratrol (C<sub>14</sub>H<sub>12</sub>O<sub>3</sub>.)</li>
+        <li>Measuring the effect on performance of funk bassplayers at temperatures exceeding 30°C (86°F), when the audience size exponentially increases (effect of 3 × 10<sup>3</sup> increasing to 3 × 10<sup>4</sup>.)</li>
+        <li> <abbr title="Hypertext Markup Language">HTML</abbr> and <abbr title="Cascading Styling Sheet">CSS</abbr> constructs for representing musical scores.</li>
+    </ol>    
+        
+    <p>   
+        So please can you provide more information on each of these subjects, including how long you'd expect the research to take, required staff and other resources, and anything else you think we'd need to know? Thanks.    
+    </p>  
+
+    <!--Added H2 tag-->
+    <h2> Exotic dance moves </h2>
+    <p>    
+        Yes, you are right! As part of my post-doctorate work, I did study exotic tribal dances. To answer your question, my favourite dances are as follows, with definitions:
+    </p>
+
+    <!--Used description lists-->    
+    <dl>
+        <dt>Polynesian chicken dance</dt>
+        <dd>
+            A little known but very influential dance dating back as far as 300<abbr title="Before Christ">BC</abbr>, a whole village would dance around in a circle like chickens, to encourage their livestock to be "fruitful".
+        </dd>
+
+        <dt>Icelandic brownian shuffle</dt>
+        <dd>
+        Before the Icelanders developed fire as a means of getting warm, they used to practice this dance, which involved huddling close together in a circle on the floor, and shuffling their bodies around in imperceptibly tiny, very rapid movements. One of my fellow students used to say that he thought this dance inspired modern styles such as Twerking.
+        </dd>
+
+        <dt>Arctic robot dance</dt>
+        <dd>
+        An interesting example of historic misinformation, English explorers in the 1960s believed to have discovered a new dance style characterized by "robotic", stilted movements, being practiced by inhabitants of Northern Alaska and Canada. Later on however it was discovered that they were just moving like this because they were really cold.
+        </dd>
+    </dl>
+
+    <!--Added a tag for hyperlink-->
+    <p>
+        For more of my research, see my <a href="www.example.com" target="_blank">exotic dance research page</a>.
+    </p> 
+    <p>
+        Yours sincerely,
+    </p>
+    <p>
+        Dr Eleanor Gaye
+    </p>   
+    
+    <!--Added i tag to italisize the important words. Also used the abbreviation tag-->
+    <p>   
+     University of Awesome motto: "Be awesome to each other." -- <i>The memoirs of Bill S Preston,<abbr title="Esquire">Esq</abbr></i> 
+    </p> 
+      
+</body>
+</html>
```

Comment:
Add a new line at the end of each file :)

[Here is why](https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b) that's needed.